db/migrate/20200515122239_create_households.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Migration to create the households table
+class CreateHouseholds < ActiveRecord::Migration[6.0]
+  # This class is generated after running "jets db:generate create_households"
+  # The command on its own will generate a migration class that initially,
+  # only creates the table. Additional columns can be added after running
+  # the create command.
+  def change
+    # Implicitly creates an id field for primary key
+    create_table :households do |t|
+      t.integer :number, null: false
```

Comment:
This column name seems vague to me. I have no idea what `number` is in the context of a household. Any chance this could have a more descriptive name?

Reply:
@nvengal I reached out to @csteele86 to inquire. This is meant to be the validation code for the household using the SafeRandom.generate_code(6) method you created. With that in mind, I'll rethink the logic on how this gets set and rename. We wanted the identification code to be on the household, correct?

Reply:
Got ya. Yeah, seems reasonable to have the identification code on household. I still think the column could have a more descriptive name. Number made me think it was related to a count or something like that.

Reply:
FYI, in the ERD, we've named this column household_number.

Reply:
There might be an issue here. Currently, we are assigning the identification_code/household_number to a user, which in the new ERD translates to the family_member. Shouldn't we keep the identification_code/household_number to the family_member or users table?

Reply:
The identification code is currently represented on the user, and in the future would make the most sense to be associated with a family_member or user. Each member should be able to have their own id code which is then able to lookup their family - not just a single code per family.

Reply:
@csteele86 @nvengal @davidpickering 

So, in conclusion, we are thinking this field makes the most sense to be included in the household_member table?

Reply:
We have the same concept on both tables. If we wanted to keep both, we could simply generate one for the household during the data migration. The question is, is there value in having both or do we just keep the household member one? @davidpickering

Reply:
Okay, make sense. I suppose for some processes I would want the household_member code and in other cases I would want the household_code. I like that, let's go with keep both. - let's do make a note though; the codes should be exclusive of each other. e.g. household code could be a 7 or 8 character version of the household_member code.